openvdb_houdini/SOP_NodeVDB.cc

```diff
@@ -720,19 +707,11 @@ OpenVDBOpFactory::setNativeName(const std::string& name)
 {
     // SideFX nodes have no native equivalent.
 #ifndef SESI_OPENVDB
-    mNativeName = name;
+    addSpareData({{"nativename", name}});
```

Comment:
I'd prefer to keep the existing solution where we update a member variable on the factory and then populate the spare data just once during factory deconstruction to avoid multiple read and write operations having to go through this dict.

Reply:
The native name will be set at most twice per operator, so the overhead is negligible.